common/src/main/scala/explore/AppMain.scala

```diff
@@ -106,6 +108,7 @@ trait AppMain extends IOApp {
       appConfig <- fetchConfig
       _         <- logger.info(s"Config: ${appConfig.show}")
       ctx       <- AppContext.from[IO](appConfig)
+      _         <- ctx.clients.odb.connect(JsonObject("Authorization" -> "Bearer ...".asJson))
```

Comment:
I guess we need to stick the token instead of `...`?

Reply:
Exactly. I just pushed a new commit with further changes to clue which allow us to specify a different token in each reconnection.